app/models/rejection.rb

```diff
@@ -11,4 +11,12 @@ class Rejection < ActiveRecord::Base
   belongs_to :visit
 
   validates :reason, inclusion: { in: REASONS }
+
+  def pvo_possible?
```

Comment:
When a prisoner has an 'enhanced' status, meaning they have above the
standard allocation of visits per month, the extra visits they have are
often exclusively for specific slots. These extra visits are known as PVOs.

Most commonly, in prisons where standard visits happen during the week,
PVOs can be used on weekends. There is no consistent pattern to this
however.

For the purposes of PVB this shouldn't matter, as the staff manage all of
that stuff through NOMIS.

On 24 November 2015 at 10:43, Paul Battley <notifications@github.com> wrote:

> In app/models/rejection.rb
> <https://github.com/ministryofjustice/prison-visits-2/pull/33#discussion_r45719590>
> :
>
> > @@ -11,4 +11,12 @@ class Rejection < ActiveRecord::Base
> >    belongs_to :visit
> >
> >    validates :reason, inclusion: { in: REASONS }
> > +
> > +  def pvo_possible?
>
> The "P" in PVO is "privileged", but it's still an opaque identifier
> without context.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ministryofjustice/prison-visits-2/pull/33/files#r45719590>
> .
>



-- 
*Kylie Mulholland*
Product Manager: Prison Visits Booking
Ministry of Justice | 102 Petty France | London | SW1H 9AJ